oio/xcute/doc/README.md

```diff
@@ -0,0 +1,246 @@
+# xcute
+
+xcute is a tool that dispatches tasks
```

Comment:
Dispatching task is the method, not the goal. I'd rather say that xcute is a tool for executing jobs in a distributed manner.